react/features/large-video/components/LargeVideo.web.js

```diff
@@ -126,6 +166,32 @@ class LargeVideo extends Component<Props> {
         );
     }
 
+    _updateLayout: () => void;
+
+    /**
+     * Refreshes the video layout to determine the dimensions of the stage view.
+     * If the filmstrip is toggled it adds CSS transition classes and removes them
+     * when the transition is done.
+     *
+     * @returns {void}
+     */
+    _updateLayout() {
+        const { _verticalFilmstripWidth, _resizableFilmstrip } = this.props;
+
+        if (_resizableFilmstrip && _verticalFilmstripWidth >= FILMSTRIP_BREAKPOINT) {
+            this._containerRef.current.classList.add('transition');
+            this._wrapperRef.current.classList.add('transition');
+            VideoLayout.refreshLayout();
+
+            setTimeout(() => {
+                this._containerRef.current.classList.remove('transition');
```

Comment:
nit. maybe check if component is still mounted by the time this executes or check this._containerRef/_wrapperRef?.current? , as if it unmounted meanwhile, there might be a crash due to the ref being destroyed.